src/axom/quest/IntersectionShaper.hpp

```diff
@@ -311,7 +311,9 @@ class IntersectionShaper : public Shaper
                      sidre::MFEMSidreDataCollection* dc)
     : Shaper(shapeSet, dc)
   {
+#if defined(AXOM_USE_RAJA) && defined(AXOM_USE_UMPIRE)
     m_free_mat_name = "free";
+#endif
```

Comment:
This change seems odd to me.

Is there a check elsewhere that requires RAJA/Umpire for this string being non-empty?

Reply:
The definition of `m_free_mat_name`, at the bottom of this file, is within `#if defined(AXOM_USE_RAJA) && defined(AXOM_USE_UMPIRE)`.  Since `m_free_mat_name` is used extensively throughout code guarded on those symbols and nowhere else (except this point), it made more sense to me to guard the initialization rather than move the definition out of its guards.